abci/client/socket_client.go

```diff
@@ -82,15 +78,12 @@ RETRY_LOOP:
 }
 
 func (cli *socketClient) OnStop() {
-	cli.BaseService.OnStop()
```

Comment:
should we also remove this function and `BaseService.OnStart()` as they don't do anything, then they can be removed elsewhere?

Reply:
These functions act as defaults and sometimes overwritten (e.g. in BaseReactor). But we can safely remove BaseService.OnStart and OnStop calls (what I did here).

Reply:
👍 ah okay